sgkit_plink/cli.py

```diff
@@ -0,0 +1,95 @@
+"""
+Command line utilities for conversion.
+"""
+import argparse
+import os
+import signal
+
+from dask.diagnostics import ProgressBar
+
+from . import read_plink
+
+
+def set_sigpipe_handler():
+    if os.name == "posix":
+        # Set signal handler for SIGPIPE to quietly kill the program.
+        signal.signal(signal.SIGPIPE, signal.SIG_DFL)
+
+
+def run_plink2sg(args):
+    # TODO add args for bim and fam seps
+    # TODO add logging and verbosity so we can write out info messages
+    # about what we're doing here.
+    ds = read_plink(args.plink_path, bim_sep=" ", fam_sep=" ")
+
+    # TODO add a command line options for this progess bar.
+    with ProgressBar():
+
+        # TODO Do the dask compute manually so that we can have
+        # more influence over the number of threads used.
+
+        # TODO add options to give more control over the zarr
+        # format used. We probably want to have the option of
+        # writing to a ZipFile anyway, so that we don't have
+        # gazillions of files.
+
```

Comment:
FWIW, here's an example that sets custom numcodec filters for Zarr: https://github.com/related-sciences/gwas-analysis/blob/master/notebooks/platform/xarray/lib/io/core.py#L60.  I can see that becoming another thing that we do often between `read_*` and `ds.to_zarr` in addition to using options for zipping files and maybe rechunking.